parallel-install/pkg/git/resolve.go

```diff
@@ -33,30 +32,14 @@ func (rl *remoteRefLister) List(repoURL string) ([]*plumbing.Reference, error) {
 
 var defaultLister refLister = &remoteRefLister{}
 
-// resolveRevision tries to convert a pseudo-revision reference (e.g. semVer, tag, PR, main, etc...) into a revision that can be checked out.
-func ResolveRevision(repo, rev string) (string, error) {
+// ResolvePRrevision tries to convert a PR into a revision that can be checked out.
+func ResolvePRrevision(repoName, rev string) (string, error) {
```

Comment:
I think this functionality should not be exported, it is an internal working of the package.

Therefore having this function is also not bringing any value. I would put this `switch` in `resolveRevision` and make the default case call `go-git`'s function.